.github/workflows/prod-test.yml

```diff
@@ -24,6 +24,23 @@ jobs:
       - name: Install Dependencies
         run: yarn install --frozen-lockfile
 
+      - name: Cache Build Results
+        # Prevent error on re-run
+        # https://github.com/actions/cache/issues/107#issuecomment-556037278
+        if: github.event_name == 'push' || github.event_name == 'pull_request'
+        id: build-cache
+        uses: actions/cache@v1
+        with:
+          path: .cache
```

Comment:
Shouldn't it be `yarn cache dir`? https://github.com/actions/cache/blob/master/examples.md#node---yarn

Reply:
You misunderstood, we have a dir [`.cache` in project root](https://github.com/prettier/prettier/blob/232661c13652d0649113aa10f0332f9934b86df4/scripts/build/build.js#L69), after building process, the result will cached there, so on later build, if cache hit, we won't need bundle this file.  Nothing related to yarn.